_overviews/scala3-book/taste-modeling.md

```diff
@@ -99,17 +162,22 @@ println(p.firstName)   // "John"
 p.lastName = "Legend"
 p.printFullName()      // "John Legend"
 ```
+{% endtab %}
+{% endtabs %}
 
 Notice that the class declaration creates a constructor:
 
+{% tabs class_2 class=tabs-scala-version %}
+{% tab 'Scala 2 and 3' for=class_2 %}
 ```scala
 // this code uses that constructor
 val p = Person("John", "Stephens")
```

Comment:
this is not valid in scala 2 without `new`

Reply:
changed.